packages/livechat/src/components/App/App.tsx

```
@@ -24,6 +25,10 @@ import SwitchDepartment from '../../routes/SwitchDepartment';
 import TriggerMessage from '../../routes/TriggerMessage';
 import type { Dispatch, StoreState } from '../../store';
 import { ScreenProvider } from '../Screen/ScreenProvider';
+import { GlobalErrorFallback } from '../../../../ui-client/src/components/errors/ErrorFallbacks';
+import { errorTrackingService } from '../../../../ui-client/src/services/ErrorTrackingService';
+import { ScreenProvider } from '../Screen/ScreenProvider';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ScreenProvider'. Consider to delete it or rename it.

'ScreenProvider' is defined here:


(lint/suspicious/noRedeclare)